src/components/ColorScheme/ColorScheme.stories.mdx

```diff
@@ -0,0 +1,69 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
+import Button from '../Button/Button';
+import ColorScheme, { schemes } from './ColorScheme';
+
+<Meta title="ColorScheme" component={ColorScheme} />
+
+# ColorScheme
+
+`ColorScheme` defines CSS color variables that can be used by other components
+to make themselves "theme-aware." A `ColorScheme` is generally rendered at the
+page level to define the color scheme of that page.
+
+Render multiple `ColorScheme`s with different `type` props to create an
+environment that responds to the user's preferences (e.g., "dark mode" /
+"light mode"):
+
+```js
+import { ColorScheme, schemes } from '@quartz/interface';
+
+function MyPage() {
+	return (
+		<>
+			<ColorScheme {...schemes.LIGHT} type="default" />
+			<ColorScheme {...schemes.DARK} type="dark" />
+			<ColorScheme {...schemes.PRINT} type="print" />
+			<h1>Hello, world!</h1>
+		</>
+	);
+}
+```
+
+Some colors are optional; note the described fallback behavior in the prop
+descriptions. Other color values are not configurable and are calculated
+automatically from base values.
+
+## Usage guidelines
+
+- **Do** supply colors in hexadecimal notation, e.g., `#000000`.
+- **Do** define official color schemes inside Prism alongside the existing `schemes`.
+- **Do** use colors from `@quartz/styles` when defining color schemes, modifying it if needed.
+- **Do** wrap `ColorScheme` in `Helmet` to hoist it to the `<head>`.
```

Comment:
Is this qz-react specific?

Reply:
well ... all apps should follow the advice to hoist it into the head. they don't have to use `react-helmet`, but ... it's kind of the only game in town. i'll clarify a tiny bit

Reply:
Awesome - sounds good. I know next to nothing about helmet, so this was more for my own education 😂 😬